termbox/src/lib.rs

```diff
@@ -405,6 +383,57 @@ impl Termbox {
             .unwrap();
         }
     }
+
+    fn send_char(&mut self, to_x: u16, to_y: u16, ch: char) {
+        // if the target cell is next to the last cell, then don't move the cursor
+        if (self.last_cursor.0 + 1) == to_x && self.last_cursor.1 == to_y {
+            write!(self.output_buffer, "{}", ch).unwrap();
+        } else {
+            goto(to_x, to_y, &mut self.output_buffer);
+            write!(self.output_buffer, "{}", ch).unwrap();
+        }
+        self.last_cursor = (to_x, to_y);
+    }
+}
+
+const LOOKUP: &[u8] = b"0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
```

Comment:
This is only used to convert numbers 0-9 to the ASCII representations so this shouldn't have letters.

Reply:
Is the lookup needed at all, considering our value range is 0-9, and our output is `b'0'-`b'9'`?

Reply:
No. It is not needed.